apps/console/src/features/identity-providers/components/forms/components/common-pluggable-component-form.tsx

```diff
@@ -60,6 +60,14 @@ export const CommonPluggableComponentForm: FunctionComponent<CommonPluggableComp
     // Used for field elements which needs to listen for any onChange events in the form.
     const [ dynamicValues, setDynamicValues ] = useState<CommonPluggableComponentInterface>(undefined);
     const [ customProperties, setCustomProperties ] = useState<string>(undefined);
+    const [ manualMode, setManualMode ] = useState<boolean>(true);
+
+    const commonKeys: string[] = [ "SelectMode", "SPEntityId", "NameIDType" ];
+
+    const enum SelectModTypes {
```

Comment:
lets fix the typo and move this to models.
```suggestion
    const enum SelectModeTypes {
```

Reply:
Fixed with a8c09e08bad85f345fff3fe7d92fa367cf58877e.